server/sonar-server/src/test/java/org/sonar/server/permission/ws/template/RemoveGroupFromTemplateActionTest.java

```diff
@@ -222,25 +187,14 @@ private void newRequest(@Nullable String groupName, @Nullable String templateKey
     request.execute();
   }
 
-  private void commit() {
-    dbSession.commit();
-  }
-
-  private GroupDto insertGroup(GroupDto groupDto) {
-    return dbClient.groupDao().insert(dbSession, groupDto);
-  }
-
-  private PermissionTemplateDto insertPermissionTemplate(PermissionTemplateDto permissionTemplate) {
-    return dbClient.permissionTemplateDao().insert(dbSession, permissionTemplate);
-  }
-
-  private void addGroupToPermissionTemplate(long permissionTemplateId, @Nullable Long groupId, String permission) {
-    dbClient.permissionTemplateDao().insertGroupPermission(dbSession, permissionTemplateId, groupId, permission);
+  private void addGroupToTemplate(PermissionTemplateDto template, @Nullable Long groupId, String permission) {
+    db.getDbClient().permissionTemplateDao().insertGroupPermission(db.getSession(), template.getId(), groupId, permission);
+    db.commit();
   }
 
   private List<String> getGroupNamesInTemplateAndPermission(long templateId, String permission) {
     PermissionQuery permissionQuery = PermissionQuery.builder().setPermission(permission).build();
-    return dbClient.permissionTemplateDao()
-      .selectGroupNamesByQueryAndTemplate(dbSession, permissionQuery, templateId);
+    return db.getDbClient().permissionTemplateDao()
```

Comment:
I don't think I agree with with writing `db.getDbClient()` and `db.getSession()` a 1000 time in test code.

We hunt code duplication in test code with private methods, XXXTesting, XXXTester and other means and we would keep on duplicating these calls?
It doesn't make sense. In addition, it's totally equivalent since `DBTester` stores a single `DbSession` and single `DbClient`. It's equivalent when `DbTester` is a `Rule` and it's not a problem when `DBTester` is a class rule as test class fields live only for a single test method call